tools/internal_ci/linux/grpc_tsan_on_foundry.sh

```diff
@@ -0,0 +1,87 @@
+Skip to content
```

Comment:
Assuming lines 1-18 are accidentally included :)

Reply:
Yeah. Fixed. :)

Reply:
I see the extras at the end of file removed, but I still see the original lines 1-18

Reply:
Ah sorry. Done.